job/server/src/main/java/alluxio/job/plan/migrate/MigrateDefinition.java

```diff
@@ -201,9 +201,9 @@ private void checkMigrateValid(MigrateConfig config, FileSystem fs) throws Excep
 
     for (Map.Entry<WorkerInfo, ArrayList<MigrateCommand>> assignment : assignments.entrySet()) {
 
-      // split the list of MigrateCommands for a given worker into JOBS_PER_WORKER.
+      // split the list of MigrateCommands for a given worker into at most equally sized JOBS_PER_WORKER lists
       List<List<MigrateCommand>> splittedCommands = Lists.partition(assignment.getValue(),
-          JOBS_PER_WORKER);
+          Math.round(assignment.getValue().size() * 1.0f / JOBS_PER_WORKER + 0.5f));
```

Comment:
Is this equivalent to using a ceiling function? Using partition causes the size of the lists to be more skewed. For example if you had 12 tasks, instead of doing [2, 2, 1, 1, 1, ...], it will do [2, 2, 2, 2, 2, 2]

Reply:
I went back to the old implementation.